src/main/webapp/src/components/entityAnalysisInfo.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/styles";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import axios from "axios";
+
+const google = window.google;
+
+const styles = (theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+  },
+});
+
+/**
+ * Displays entity analysis component for song lyrics.
+ */
+class EntityAnalysisInfo extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      entityAnalysisInfo: [],
+      isLoading: false,
+      error: null,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ isLoading: true });
+
+    axios
+      .post(
+        "/entity",
+        {},
+        {
+          headers: {
+            "Content-Type": "application/json",
+            "Access-Control-Allow-Origin": "*",
+          },
+          params: {
+            lyrics: this.props.lyrics,
+          },
+        }
+      )
+      .then((result) => result.data)
+      .then((entityAnalysisInfo) =>
+        this.setState({
+          entityAnalysisInfo: entityAnalysisInfo,
+          isLoading: false,
+        })
+      )
+      .catch((error) =>
+        this.setState({
+          error,
+          isLoading: false,
+        })
+      );
+  }
+
+  render() {
+    const classes = this.props.classes;
+
+    if (this.state.error) {
+      return (
+        <div>
+          <p>{this.state.error.message}</p>
+        </div>
+      );
+    }
+
+    if (this.state.isLoading) {
+      return (
+        <div>
+          <CircularProgress style={{ color: "black" }} />;
+        </div>
+      );
+    }
+
+    if (this.state.entityAnalysisInfo.length == 0) {
+      return <p>No entities was found.</p>;
```

Comment:
This should be rephrased.
Maybe
`No entities were found` or just `No entities found`